defactor-com/ui-kit
# feat(table): add param empty icon

## Add param empty icon

### What does this PR do?

- Resolve #181 

### Steps to test

1. Run project
2. Check changes

Now:
![imagen](https://github.com/defactor-com/ui-kit/assets/49013556/13d07c2a-8ce5-4605-bd18-5a150c9fdb87)


#### CheckList
